modules/stanford_news_extras/README.md

```diff
@@ -9,7 +9,25 @@ Installation
 Install this module like any other module. [See Drupal Documentation](https://drupal.org/documentation/install/modules-themes/modules-7)
 
 Configuration
----
+-------------
+Current approach to installing all the things for importing news from SoE
+
+1. Install and enable the module stanford_feeds_helper (https://github.com/SU-SWS/stanford_feeds_helper.git)
```

Comment:
This seems to be specific to the SOE website. Can we change this to a more generic set of instructions.